requirements.txt

```
@@ -1,8 +1,8 @@
 # ocrd includes opencv, numpy, shapely, click
-ocrd >= 3.0.0b4
+ocrd >= 3.1.0
```

```suggestion
ocrd >= 3.3.0
```
For consistency with the docker base image.